### ISSUE TITLE
Custom Embedding Model

### ISSUE THREAD

**Issue by walbercardoso** [safevideo/autollm#146]
Can i use a open Embedding Model instead openAI?

**Comment by fcakyon** [comment]
Yes for instance, you can use huggingface embedding models. Check the comment in this line for example usage: https://github.com/safevideo/autollm/blob/e3ce9884b36b037327146897fc2252eeb7d58c5c/examples/configs/config.example.yaml#L16
